### PR TITLE
Prepare version 2.2.0crp

### PR DESCRIPTION
**Pull request by slizzered**
- version file updated
- `CHANGELOG.md` updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
 Change Log / Release Log for mallocMC
 ================================================================
 
+2.2.0crp
+-------------
+**Date:** 2015-09-24
```
Comment (ax3l) [inline CHANGELOG.md]: oh oh, can you pls update? (sry!)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+We closed all issues documented in
+[Milestone *2.2.0crp: Stabilizing the release*](https://github.com/ComputationalRadiationPhysics/mallocMC/issues?milestone=5&state=closed)
+
+### Changes to mallocMC 2.1.0crp
+
+**Features**
+ - the interface now provides the host function `HeapInfoVector getHeapLocations()` to obtain information about the location and size of existing mallocMC-heaps. #86
+
+**Bug fixes**
+ - the function `getAvailableSlots` was always required in the policy classes, although the implementations might not provide it. #89
```
Comment (ax3l) [inline CHANGELOG.md]: end on `.`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,29 @@
...
+**Date:** 2015-09-24
+
+This release fixes some minor bugs that occured after the release of 2.1.0crp, adds some documentation and improves the interoperability with other projects and build systems.
+We closed all issues documented in
+[Milestone *2.2.0crp: Stabilizing the release*](https://github.com/ComputationalRadiationPhysics/mallocMC/issues?milestone=5&state=closed)
+
+### Changes to mallocMC 2.1.0crp
+
+**Features**
+ - the interface now provides the host function `HeapInfoVector getHeapLocations()` to obtain information about the location and size of existing mallocMC-heaps. #86
```
Comment (ax3l) [inline CHANGELOG.md]: shoud not end on `.`?

**Comment by slizzered** [comment]
updated!

**Comment by ax3l** [comment]
thx! :sparkles:

**Pull request merged**